packages/fast-form-generator-react/app/index.tsx

```diff
@@ -52,48 +59,73 @@ export default class App extends React.Component<{}, IAppState> {
+            currentControlledComponentSchema: testComponents.textField.schema,
+            currentControlledComponentData: getExample(testComponents.textField.schema),
+            currentControlledComponentOrderByPropertyNames: void(0),
+            currentControlledComponentAttributeAssignment: void(0),
+            currentUncontrolledComponent: testComponents.textField.component,
+            currentUncontrolledComponentSchema: testComponents.textField.schema,
+            currentUncontrolledComponentData: getExample(testComponents.textField.schema),
+            currentUncontrolledComponentOrderByPropertyNames: void(0),
+            currentUncontrolledComponentAttributeAssignment: void(0),
+            controlledOnChange: this.controlledOnChange,
+            uncontrolledOnChange: this.uncontrolledOnChange,
             showExtendedControls: false,
-            schemaLocation: void(0),
-            dataLocation: void(0)
+            schemaLocation: "",
+            dataLocation: ""
         };
     }
 
     public render(): JSX.Element {
         return (
             <DesignSystemProvider designSystem={designSystemDefaults}>
-                <div>
-                    <div
-                        style={{width: "300px", minHeight: "100vh", padding: "0 8px", background: "rgb(244, 245, 246)", float: "left"}}
-                    >
-                        <Form {...this.coerceFormProps()} />
+                <div style={{float: "left", width: "50%"}}>
+                    <div>
+                        <div
+                            style={{width: "300px", minHeight: "100vh", padding: "0 8px", background: "rgb(244, 245, 246)", float: "left", display: "inline-block"}}
+                        >
+                            <Form {...this.coerceControlledFormProps()} />
+                        </div>
+                        <div style={{display: "inline-block"}}>
+                            <h2>Controlled</h2>
+                            <div>
+                                <select onChange={this.handleControlledComponentUpdate}>
+                                    {this.getComponentOptions()}
+                                </select>
+                            </div>
+                            <this.state.currentControlledComponent
+                                {...this.state.currentControlledComponentData}
+                            />
+                        </div>
                     </div>
+                </div>
+                <div style={{float: "left", width: "50%"}}>
                     <div>
-                        <div>
-                            <select onChange={this.handleComponentUpdate}>
-                                {this.getComponentOptions()}
-                            </select>
+                        <div
+                            style={{width: "300px", minHeight: "100vh", padding: "0 8px", background: "rgb(244, 245, 246)", float: "left", display: "inline-block"}}
```

Comment:
I think tests are failing due to max-line-length issues here and on line 94. Can we return this from a function or just define it as a constant and pass it here? Function will probably solve both CircleCI and CodeClimate issues

Reply:
Thanks, addressed.